Formula/dscanner.rb

```diff
@@ -17,7 +17,7 @@ class Dscanner < Formula
   depends_on "dmd" => :build
 
   def install
-    system "make", "dmdbuild"
+    system "make", "all"
```

Comment:
or can just do
```suggestion
    system "make"
```
as `all` is default target.

all it does is `make bin/dscanner` with default `DC=dmd`.

Looks like changing `dmd` to something else is done via `DC=...` rather than dedicated targets.